### PR TITLE
AWS migration

### PR DESCRIPTION
**Pull request by justinanderson**
Makes `vagrant up` build whole site that matches the production environment.
- Uses Nginx and Gunicorn instead of Python's runserver and stunnel.
- Adds several helper scripts to hopefully ease the learning curve on Docker.
- Generates a self-signed cert so HTTPS works out of the box on new setups.
- Changes documentation to say that Vagrant is the recommended way of working with DataHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-*Note: The project is under development. It is not ready for deployment yet.*
+*Note: This project is under development. It is not yet ready for production use.*
```
Comment (RogerTangos) [inline README.md]: nice.

**Comment by RogerTangos** [comment]
Everything looks in place to me. Thank you for updating the documentation. My main reservation is that we're not prompting the user to update their ```SECRET_KEY`` in settings.py. If we're advertising that all they need to do is clone and vagrant up, it seems wise to ask them to provide some entropy.

That aside, seems good. Is that a change you'd like to make, or should we leave it for the next pull request?

**Comment by justinanderson** [comment]
@RogerTangos Good point. I'll add a SECRET_KEY generator step.

**Comment by RogerTangos** [comment]
Thanks!

**Comment by justinanderson** [comment]
Should be all set now.

**Comment by RogerTangos** [comment]
Looks good. Thanks for storing the key in a seperate file.

**Pull request merged**